cfme/cloud/availability_zone.py

```diff
@@ -227,7 +227,9 @@ def step(self, *args, **kwargs):
 
 @navigator.register(AvailabilityZone, "Utilization")
 class Utilization(CFMENavigateStep):
-    VIEW = AzoneCloudUtilizationView
+    @property
+    def VIEW(self):     # noqa
+        return self.obj.provider.azone_utilization_view
```

Comment:
This needs to be defined (defaulted) in BaseProvider, not just in EC2Provider

Reply:
The default view is AzoneCloudUtilizationView defined in cfme/common/candu_views.

All the views specific to Availability zones are defined in cfme/cloud/availability_zone.py and availability_zone.py  has been updated so that it defaults to AzoneCloudUtilizationView if a provider specific view is not defined.